framework/doc/content/source/meshgenerators/TransfiniteMeshGenerator.md

```diff
@@ -0,0 +1,70 @@
+# TransfiniteMeshGenerator
+
+The `TransfiniteMeshGenerator` produces two-dimensional meshes from a set of 4 corner vertices by interpolating the edges into the inner domain. The resulting mesh  consists of QUAD4 elements of aspect ratio and skewness determined by the points on the boundary edges.
+
+!syntax description /Mesh/TransfiniteMeshGenerator
+
+!media large_media/framework/meshgenerators/quad_convention.png style=float:right;width:20%;margin-left:2%; caption=Fig. 1: Convention for the corner vertices order.
+
+
+## Overview
+
+&nbsp;
+
+Given a set of 4 corners the TransfiniteMeshGenerator constructs by default the straight edges that connect the vertices and fills up the interior with straight lines. The test, [quadrilater_generator.i](test/tests/meshgenerators/transfinite_generator/quadrilater_generator.i),
+illustrates the use of the `TransfiniteMeshGenerator` object to construct general quadrilaterals as in Fig. 2. By default the distribution of points along edges is considered equidistant, unless specified otherwise by the user using the parameters `bias_x` or `bias_y`. The core of the algorithm interpolates/fills up the inner domain for any edge type, curved, parametrized, discrete. The current implementation considers all these cases with special attention given to arcs of a circle which are widely encountered in scientific computing applications. 
```

Comment:
I used the same ratio and bias convention as for the other generators. I do agree this is not extensively explained in the documentation but I think it should be done the same across all generators, not only this one. I think explaining it for this would shadow the other generator tyoes.

Reply:
Ha!  I didn't realize this was preexisting bad documentation.  No need to fix it here specifically, then.